packages/typespec-vscode/snippets.json

```diff
@@ -0,0 +1,37 @@
+{
+  "Alias": {
+    "prefix": "alias",
+    "body": "alias ${1:AliasName} = ${0:Expression};",
+    "description": "Alias definition"
+  },
+  "Enum": {
+    "prefix": "enum",
+    "body": "enum ${1:EnumName} {\n  ${0:Value},\n}",
+    "description": "Enum definition"
+  },
+  "Interface": {
+    "prefix": "interface",
+    "body": "interface ${1:InterfaceName} {\n  ${2:member}(${3:parameters}): ${0:ReturnType};\n}",
+    "description": "Interface definition"
+  },
+  "Model": {
+    "prefix": "model",
+    "body": "model ${1:ModelName} {\n  ${2:propertyName}: ${0:propertyType};\n}",
+    "description": "Model definition"
+  },
+  "Namespace": {
+    "prefix": "namespace",
+    "body": "namespace ${1:NamespaceName} {\n  ${0:member}\n}",
```

Comment:
If we dont have the ${0:member} but let the cursor stop at this position when user finishes $1, will it be the same user experience as current one? Because it's not possible for user to finish the member as part of the snippet, maybe it makes more sense just left the cursor there for user to continue their job instead of making it a part of the snippet. for example, if you check the 'function' snippet of typescript, it doesn't have a placeholder as implementation.

Reply:
I changed to behavior to be consistent with typescript, e.g. `$TM_SELECTED_TEXT$0`, see https://github.com/microsoft/vscode/blob/75caed98f2ffb9e0657001af4cd0aee5bb938a84/extensions/typescript-basics/snippets/typescript.code-snippets#L176C7-L176C26